lib/app_profiler/backend/vernier.rb

```diff
@@ -0,0 +1,104 @@
+# frozen_string_literal: true
+
+gem("vernier", ">= 0.4.0")
+require "vernier"
+
+module AppProfiler
+  module Backend
+    class Vernier < Base
+      NAME = "vernier"
```

Comment:
So personally, I think we should be using symbols here. It will allocate less memory, and it is easier to type when specifying for configuration. Usually, config values are symbols over strings too. When assigning the backend, you can just cast via to_sym.